### PR TITLE
Refactor PR 395

### PR DESCRIPTION
**Pull request by aminlatifi**
Refactored seed token add migration

### CODE REVIEW

File: src/seedToken-ormconfig.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  password: process.env.TYPEORM_DATABASE_PASSWORD,
+  database: process.env.TYPEORM_DATABASE_NAME,
+  entities,
+  migrations: ['migration/1646303882607-seedTokes.ts'],
+  cli: {
+    migrationsDir: 'migration',
+  },
+};
+
+export = ormConfig;
```
Comment (mohammadranjbarz) [inline src/seedToken-ormconfig.ts:24]: @aminlatifi Do we need this file?

Comment (aminlatifi) [reply src/seedToken-ormconfig.ts:24]: I made it to test token seed migration individually. We can remove it or keep it as an example for next migration test

Comment (mohammadranjbarz) [reply src/seedToken-ormconfig.ts:24]: Yeah you are right, it's not bad to have it

**Review by mohammadranjbarz — APPROVED** [review]
@aminlatifi  Thanks Amin, the improvements are huge, I learned from your changes, 
I ran your migrations locally and tests passed, and codes seems ok, I will merge it now

**Pull request merged**